rest-api/dao/participant_summary_dao.py

```diff
@@ -215,39 +260,30 @@ def make_query_filter(self, field_name, value):
   def update_from_biobank_stored_samples(self, participant_id=None):
     """Rewrites sample-related summary data. Call this after updating BiobankStoredSamples.
     If participant_id is provided, only that participant will have their summary updated."""
-    baseline_tests_sql, baseline_tests_params = get_sql_and_params_for_array(
-        config.getSettingList(config.BASELINE_SAMPLE_TEST_CODES), 'baseline')
-    dna_tests_sql, dna_tests_params = get_sql_and_params_for_array(
-        config.getSettingList(config.DNA_SAMPLE_TEST_CODES), 'dna')
-    sample_sql, sample_params = _get_sample_sql_and_params()
-    sql = """
+    now = clock.CLOCK.now()
+    sample_sql, sample_params = _get_sample_sql_and_params(now)
+
+    baseline_tests_sql, baseline_tests_params = _get_baseline_sql_and_params()
+    dna_tests_sql, dna_tests_params = _get_dna_isolates_sql_and_params()
+
+    counts_sql = """
     UPDATE
```

Comment:
So I was wondering whether we need to do something similar for  _ENROLLMENT_STATUS_SQL. This would only be necessary when we change enrollment status criteria... but I think we are going to do that in future. 

If you don't want to do it in this PR, sounds fine, but let's file an issue to track it in future.

Reply:
Not pressing, but please file an issue.

Reply:
Done - DA-631